pymfy/api/devices/roller_shutter.py

```diff
@@ -15,11 +15,13 @@ def set_position(self, value: int, low_speed: Optional[int] = False) -> None:
         command = Command(command_name, Parameter("position", value))
         self.send_command(command)
 
-    def close(self) -> None:
-        self.send_command(Command("close"))
+    def close(self, low_speed: Optional[int] = False) -> None:
+        command = Command("position_low_speed", Parameter("position", 0) if low_speed else "close")
```

Comment:
0 means opened for Somfy and 100 closed: https://github.com/tetienne/somfy-open-api/blob/8fa69fccef71794e398b97e65fb6430104badf18/pymfy/api/devices/roller_shutter.py#L33

Reply:
Thanks, fixed.